webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({can_create}) => {
```

Comment:
Looks like this should be:
```suggestion
export const WelcomeArchitecture = ({ canCreate }) => {
```
and you'll also need to change it in the erb and in the other components